README.reproducible.md

```diff
@@ -0,0 +1,167 @@
+#Armory Reproducible Build Documentation
+
+##Introduction
+
+Gitian is just one method for achieving reproducible builds. Armory uses Gitian
+for OS X and Windows, but uses the Debian Reproducible Build Toolchain for
+Debian packages. Armory does not support the RPM builds, but provides
+instructions for doing RPM builds.
+
+##Gitian
+
+Gitian is the build system that Bitcoin Core uses for reproducible builds.
+Armory has adopted it for OS X and Windows reproducible builds. This guide
+assumes you are using KVM as the virtualization technology to make the VMs
+that Gitian uses to build in, but gitian-tools also supports LXC.
+
+The following steps will allow you to build Armory reproducibly using Gitian.
+
+1. Obtain gitian-tools from [Devrandom's GitHub][Devrandom's GitHub].
+2. Run `bin/make-base-vm --suite trusty`. Armory uses Ubuntu Trusty as the OS
+   of the VM to build in. Add `--lxc` if using LXC.
+3. If using LXC, run `export USE_LXC=1`.
+4. Run `bin/gbuild --commit BitcoinArmory=autotools-gitian ../BitcoinArmory/gitian-descriptors/bitcoin-armory-osx.yml`. The commit option is optional and
+   specifies a Git tag, branch name, or commit hash. By default it is master.
+   The command ends with the path to the appropriate YAML file in the Armory
+   source tree. In this case, the OS X Gitian descriptor is used to indicate
+   that we want to build for OS X.
+5. TODO: Include instructions about signing and verifying.
```

Comment:
Sorry. Just now noticed that this was included. This line is fine for now but it should probably be updated once PR #308 is ready to go.